warrior/WarriorCore/project_driver.py

```diff
@@ -217,21 +261,27 @@ def execute_project(project_filepath, auto_defects, jiraproj, res_startdir, logs
                     location=os.path.dirname(testsuite_path),
                     name=testsuite_nameonly, timestamp=tmp_timestamp,
                     **pj_junit_object.init_arg())
-                pj_junit_object.update_attr("status", "SKIPPED", "ts", tmp_timestamp)
-                pj_junit_object.update_attr("skipped", "1", "pj", tmp_timestamp)
-                pj_junit_object.update_count("suites", "1", "pj", tmp_timestamp)
-                data_repository['testsuite_%d_result'%suite_cntr] = "SKIP"
-                # pj_junit_object.add_testcase_message(tmp_timestamp, "skipped")
-                pj_junit_object.update_attr("impact", impact_dict.get(testsuite_impact.upper()),
+                pj_junit_object.update_attr("status", "SKIPPED", "ts",
+                                            tmp_timestamp)
+                pj_junit_object.update_attr("skipped", "1", "pj",
+                                            tmp_timestamp)
+                pj_junit_object.update_count("suites", "1", "pj",
+                                             tmp_timestamp)
+                data_repository['testsuite_{}_result'.format(suite_cntr)] = "SKIP"
+                # pj_junit_object.add_testcase_message(tmp_timestamp,
+                # "skipped")
```

Comment:
Hi @Annam16 I believe this part is complained only by default pylintrc, please use our project pylintrc file to make format change